lib/db/DB.ts

```diff
@@ -6,6 +6,7 @@ import Bluebird from 'bluebird';
 
 import Logger from '../Logger';
 import { db } from '../types';
+import P2PRepository from '../p2p/P2PRepository';
```

Comment:
This creates a circular dependency which we've been trying to avoid. Just do `Host.bulkCreate`.